app/init.php

```diff
@@ -1796,6 +1811,27 @@ function getDevice(string $root, string $connection = ''): Device
     return [];
 });
 
+App::setResource('devKey', function ($request, $project, $dbForConsole) {
+    $devKey = $request->getHeader('x-appwrite-development-key', '');
```

Comment:
We should allow to set it in URL param too. Like you can do `x-appwrite-project` header, but also `?project=` search param

Reply:
Updated.